libnmstate/netapplier.py

```diff
@@ -156,7 +157,18 @@ def _apply_ifaces_state(
                     con_profiles=ifaces_add_configs + ifaces_edit_configs,
                 )
             if verify_change:
-                _verify_change(desired_state)
+                attempts_left = 5
+                retry = True
+                while(retry):
+                    try:
+                        _verify_change(desired_state)
+                        retry = False
+                    except NmstateVerificationError:
+                        time.sleep(1)
```

Comment:
The sleep should happen after the potential raise or before the verify change. Here it just delays the execution while nothing will change.